JSDemos/Demos/DataGrid/PDFHeaderAndFooter/description.md

```diff
@@ -1,3 +1,11 @@
-<a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library allows you to customize PDF files and write custom texts and draw custom content. This demo uses this functionality to add a header (a title before exported data) and a footer (a note after exported data).
+The DataGrid allows you to customize a header and a footer in the exported PDF file. Follow the steps below to configure a header/footer:
 
-Review the code in the promise that is executed after **exportDataGrid(options)** method is called.
+1. Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method.
+
+2. Configure the following properties:    
```

Comment:
Maybe you should say "API" instead of "properties", because the second item refers to a function.

```suggestion
2. Use the following API to specify header/footer layout settings:    
```